bin/v-delete-sys-quota

```diff
@@ -41,26 +41,30 @@ if [ -n "$(echo $opt | grep $fnd)" ]; then
 fi
 
 # Disabling group and user quota
-quotaoff=$(which --skip-alias --skip-functions quotaoff 2> /dev/null)
-if [ $? -eq 0 ]; then
-	if [ -n "$(quotaon -pa | grep " $mnt " | grep 'user\|group' | grep 'is on')" ]; then
-		$quotaoff $mnt
+if quotaon="$(type -P quotaon 2>/dev/null)" && quotaoff="$(type -P quotaoff 2>/dev/null)"; then
+        if "${quotaon}" -pa | grep " $mnt " | grep 'user\|group' | grep -q 'is on' &>/dev/null; then
+		"$quotaoff" "$mnt"
 	fi
 fi
 
 # Deleting v1 + v2 group and user quota index
-for idx in $(echo 'quota.user quota.group aquota.user aquota.group'); do
+for idx in quota.user quota.group aquota.user aquota.group; do
 	[ -e "$mnt/$idx" ] && rm -f "$mnt/$idx"
 done
 
-# Deleting cron job
-rm -f /etc/cron.daily/quotacheck
+# Deleting cron job and forcequotacheck
+if [[ -f "/etc/cron.daily/quotacheck" ]]; then
+        rm -f "/etc/cron.daily/quotacheck"
+fi
+if [[ -f "/forcequotacheck" ]]; then
+        rm -f "/forcequotacheck"
+fi
 
 # Updating hestia.conf value
-if [ -z "$(grep DISK_QUOTA $HESTIA/conf/hestia.conf)" ]; then
-	echo "DISK_QUOTA='no'" >> $HESTIA/conf/hestia.conf
+if ! grep -q DISK_QUOTA "$HESTIA/conf/hestia.conf"; then
+	echo "DISK_QUOTA='no'" >> "$HESTIA/conf/hestia.conf"
 else
-	sed -i "s/DISK_QUOTA=.*/DISK_QUOTA='no'/g" $HESTIA/conf/hestia.conf
+	sed -i "s/DISK_QUOTA=.*/DISK_QUOTA='no'/g" "$HESTIA/conf/hestia.conf"
```

Comment:
$BIN/v-change-sys-config-value "DISK_QUOTA" "no"

Saves the check and make it consistent with rest of code...

We should also update line 83-87 in v-add-sys-quota